src/Utility.js

```diff
@@ -0,0 +1,36 @@
+class Utility {
+  constructor(dataSet, userID) {
+    this.dataSet = dataSet;
+    this.userID = userID;
+    this.singleUserData = this.dataSet.filter(stat => stat.userID === this.userID);
+  }
+
+  returnWeekDatesOnly() {
+    return [...this.singleUserData].splice(-7 * week, 7).map(day => day.date);
+  }
```

Comment:
I think we will need to include a 'week' parameter here since the method is referencing it.